pkg/kubelet/kubelet.go

```diff
@@ -2830,6 +2835,7 @@ func (kl *Kubelet) handlePodResourcesResize(pod *v1.Pod) (*v1.Pod, error) {
 	if resizeStatus != "" {
 		// Save resize decision to checkpoint
 		if err := kl.statusManager.SetPodResizeStatus(allocatedPod.UID, resizeStatus); err != nil {
+			kl.recorder.Eventf(allocatedPod, v1.EventTypeWarning, events.ResizeError, "Resize pod %s failed: %v", klog.KObj(allocatedPod), err)
```

Comment:
This error name is too broad. Please name this as ResizeCheckpointError, and it can be used here as well as at SetPodAllocation failure above.